### PR TITLE
Add The Sims 2 - Ultimate Collection

### PR DESCRIPTION
**Pull request by nunnsy**
Wasn't sure how much information to include in the public documentation, but even with all sorts of d3d9 tricks applied, whether in the install folder, or in the system folders, it still results in graphic related errors (first run has a slightly different log output to subsequent runs).

### CODE REVIEW

File: src/game-support/the-sims-2.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# The Sims 2 (Ultimate Collection)
+<!-- script:Aliases [] -->
+
+{{#template ../templates/rating.md status=Garbage installs=Yes opens=No}}
+
+> [!WARNING]
+> The Sims 2 is reliant on DirectX 9, which is largely unsupported by Whisky at this stage
```
Comment (mattmaniak) [inline src/game-support/the-sims-2.md:7]: A little bit pedantic, but leave a dot at the end of the phrase and it is fine.

**Review by mattmaniak — CHANGES_REQUESTED** [review]

File: src/game-support/the-sims-2.md (hunk 1 above)
Comment (nunnsy) [inline src/game-support/the-sims-2.md:7]:
```suggestion
> The Sims 2 is reliant on DirectX 9, which is largely unsupported by Whisky at this stage.
```

File: src/game-support/the-sims-2.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+> [!NOTE]
+> This game is not available to purchase and the Windows version is considered abandonware. This guide uses the open source installer [The Sims 2 Starter Pack](https://github.com/voicemxil/TS2-Starter-Pack)
```
Comment (nunnsy) [inline src/game-support/the-sims-2.md:10]:
```suggestion
> This game is not available to purchase and the Windows version is considered abandonware. This guide uses the open source installer [The Sims 2 Starter Pack](https://github.com/voicemxil/TS2-Starter-Pack).
```

File: src/game-support/the-sims-2.md (hunk 1 above)
Comment (nunnsy) [reply src/game-support/the-sims-2.md]: Updated both remarks, cheers

**Review by mattmaniak — APPROVED** [review]

**Pull request merged**